Meshtastic/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,315 @@
+				}
+			} label: {
+				Text("Configure notification permissions")
+					.frame(maxWidth: .infinity)
+			}
+			.buttonBorderShape(.capsule)
+			.controlSize(.large)
+			.padding()
+			.buttonStyle(.borderedProminent)
+		}
+	}
+
+	var locationView: some View {
+		VStack {
+			VStack {
+				Text("Phone Location")
+					.font(.largeTitle.bold())
+					.multilineTextAlignment(.center)
+					.fixedSize(horizontal: false, vertical: true)
+			}
+			VStack(alignment: .leading, spacing: 16) {
+				Text("Meshtastic uses your phone's location to enable a number of features. You can update your location permissions at any time from Settings > App Settings > Open Settings.")
+					.font(.body.bold())
+					.multilineTextAlignment(.center)
+					.fixedSize(horizontal: false, vertical: true)
+				makeRow(
+					icon: "location",
+					title: "Share Location",
+					subtitle: "Use your phone GPS to send locations to your node to instead of using a hardware GPS on your node."
+				)
+				makeRow(
+					icon: "lines.measurement.horizontal",
+					title: "Distance Measurements",
+					subtitle: "Used to display the distance between your phone and other Meshtastic nodes where positions are available."
+				)
+				makeRow(
+					icon: "line.3.horizontal.decrease.circle",
+					title: "Distance Filters",
+					subtitle: "Filter the node list and mesh map based on proximity to your phone."
+				)
+				makeRow(
+					icon: "mappin",
+					title: "Mesh Map Location",
+					subtitle: "Enables the blue location dot for your phone in the mesh map."
+				)
+			}
+			.padding()
+			Spacer()
+			if LocationHelper.shared.locationManager.authorizationStatus != .notDetermined {
+				Button {
+					Task {
+						await goToNextStep(after: .location)
+					}
+				} label: {
+					Text("Continue to next step")
+						.frame(maxWidth: .infinity)
+				}
+				.padding()
+				.buttonBorderShape(.capsule)
+				.controlSize(.large)
+				.padding()
+				.buttonStyle(.borderedProminent)
+			}
+		}
+	}
+
+	var mqttView: some View {
+		VStack {
+			VStack {
+				Text("MQTT")
+					.font(.largeTitle.bold())
+					.multilineTextAlignment(.center)
+					.fixedSize(horizontal: false, vertical: true)
+			}
+			Spacer()
+			Button {
+				Task {
+
+				}
+			} label: {
+				Text("Enable MQTT")
+					.frame(maxWidth: .infinity)
+			}
+			.padding()
+			.padding()
+			.buttonBorderShape(.capsule)
+			.controlSize(.large)
+			.padding()
+			.buttonStyle(.borderedProminent)
+
+			Button {
+				dismiss()
+			} label: {
+				Text("Set up later")
+					.frame(maxWidth: .infinity)
+			}
+		}
+	}
+
+    var body: some View {
+		NavigationStack(path: $navigationPath) {
+			welcomeView
+				.navigationDestination(for: SetupGuide.self) { guide in
+					switch guide {
+					case .notifications:
+						notificationView
+					case .location:
+						locationView
+					case .mqtt:
+						mqttView
+					}
+				}
+		}
+		.toolbar(.hidden)
+	}
+
+	@ViewBuilder
+	func makeRow(
```

Comment:
This `makeRow` ViewBuilder takes strings, but these are not localized. To ensure this will be localized these should be changed to use [`LocalizedStringResource`](https://developer.apple.com/documentation/foundation/localizedstringresource).

```suggestion
	func makeRow(
	    iconSystemName: String,
	    title: LocalizedStringResource,
	    subtitle: LocalizedStringResource
	  ) -> some View { .. }
```